ci-operator/step-registry/openshift/microshift/infra/iso-build/OWNERS

```diff
@@ -1,19 +1,4 @@
-# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
-# Fetched from https://github.com/openshift/microshift root OWNERS
-# If the repo had OWNERS_ALIASES then the aliases were expanded
-# Logins who are not members of 'openshift' organization were filtered out
-# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
-
 approvers:
-- copejon
-- dhellmann
-- ggiguash
-- pacevedom
-- pmtk
-options: {}
```

Comment:
Do we need to remove this?

Reply:
I checked other owner files and only few of them contain these empty options. 
```
$ pwd
/home/ggiguash/Projects/openshift-release/ci-operator/step-registry/openshift
$ find . -name OWNERS -exec grep -l options: {} \; | wc -l
34
$ find . -name OWNERS -exec grep -L options: {} \; | wc -l
329
```

Reply:
I do not know whether it's necessary, but it looked like a copy-paste to me.

Reply:
I can return those back if you think they are necessary.